spec/enrichments/tsdoc_v2/combined.json

```diff
@@ -98272,7 +98272,7 @@
                           "kindString": "Parameter",
                           "flags": {},
                           "comment": {
-                            "shortText": "The relative file path. Should be of the format `folder/subfolder/filename.png`. The bucket must already exist before attempting to upload."
+                            "shortText": "The relative file path. Should be of the format `folder/subfolder/filename.png`. The bucket must already exist before attempting to update."
```

Comment:
Is this "should" (recommended) or "must" (requirement)?

Reply:
its a requirement Danny